user/src/App.jsx

```diff
@@ -1,7 +1,15 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom'
+import Footer from './components/footer/Footer';
+import Header from './components/header/Header';
+
 function App() {
   return (
-    <div className="App container-fluid">
-        
+    <div className="App">
+      <Router>
+        <Header />
+        <Footer />
+      </Router>
```

Comment:
```suggestion
      <Header />
      <Router>
      </Router>
      <Footer />
```
Bởi vì Header + Footer là trang nào cũng có. Những thứ cần thay đổi là ở bên trong Router cơ